app/src/components/main/modal/modalLayout/modalContent/sectionHeader/sectionHeader.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './sectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+export const SectionHeader = ({ text, error }) => (
```

Comment:
If it's going to be used outside of a modal, it should be placed somewhere else.
`components/main` maybe?

Reply:
Moved